src/Core/src/Hosting/Internal/MauiServiceCollection.cs

```diff
@@ -13,7 +13,7 @@ class MauiServiceCollection : IMauiServiceCollection
 
 		public int Count => _descriptors.Count;
 
-		public bool IsReadOnly => false;
+		public static bool IsReadOnly => false;
```

Comment:
`D:\a\_work\1\s\src\Core\src\Hosting\Internal\MauiServiceCollection.cs(9,32): error CS0736: 'MauiServiceCollection' does not implement instance interface member 'ICollection<ServiceDescriptor>.IsReadOnly'. 'MauiServiceCollection.IsReadOnly' cannot implement the interface member because it is static. [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=netstandard2.0]`

The changes to static is breaking the build.